apps/src/studio/blocks.js

```diff
@@ -2060,11 +2060,22 @@ exports.install = function (blockly, blockInstallOptions) {
         this.appendDummyInput()
           .appendTitle(msg.waitSeconds());
       } else {
-        var dropdown = new blockly.FieldDropdown(this.VALUES);
-        dropdown.setValue(this.VALUES[2][1]);  // default to half second
+        if (isK1) {
+          let dropdown = new blockly.FieldDropdown([1, 2, 3, 4, 5].map(
+                (val) => [val.toString(), (val * 1000).toString()]));
```

Comment:
Mainly because those looked like too much text for a K-1 block:
![dropdown](https://cloud.githubusercontent.com/assets/1070243/17343576/3696c11e-58b3-11e6-84a8-887928ec05c1.png)